source/model.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Created by Ksenia Annenko on 24.02.2017.
+ */
+
+/********************** Model **********************/
+"use strict";
+function Field() {
+    let self = this;
+    let cellsCount = 9;
+    this.fieldCells = [];
+
+    this.updateCell = function (cellNum, symbol) {
+        return self.fieldCells[cellNum] != null ? false : self.fieldCells[cellNum] = symbol;
+    };
+
+    this.getEmptyCells = function () {
+        let emptyCells = [];
+        for (let i = 0; i < cellsCount; i++) {
+            if (self.fieldCells[i] == null) {
+                emptyCells.push(i);
+            }
+        }
+        return emptyCells;
+    };
+
+    this.areCellsAvailable = function () {
+        return self.getEmptyCells().length > 0;
+    }
+}
+
+function WinnerChecker(field) {
+    let winCombinations = [
+        [0, 1, 2], [3, 4, 5], [6, 7, 8],
+        [0, 3, 6], [1, 4, 7], [2, 5, 8],
+        [0, 4, 8], [2, 4, 6]
+    ];
+    let currentWinCombination = null;
```

Comment:
Can this be moved to the getWinner function? It seems the class doesn't use this var anywhere else.